packages/react-html/src/hooks.ts

```diff
@@ -58,7 +58,7 @@ export function useClientDomEffect(
     () => {
       perform(manager);
     },
-    [manager, ...inputs],
```

Comment:
the react-hooks rule does not like this spread.

Reply:
Do you know if there is an issue for this on the react repo?